src/main/scala/mesosphere/marathon/api/v2/json/GroupUpdate.scala

```diff
@@ -40,7 +41,7 @@ case class GroupUpdate(
         .map(app => app.id -> app)(collection.breakOut)
 
     val effectiveDependencies = dependencies.fold(current.dependencies)(_.map(_.canonicalPath(current.id)))
-    Group(current.id, effectiveApps, effectiveGroups, effectiveDependencies, timestamp)
+    Group(current.id, effectiveApps, current.pods, effectiveGroups, effectiveDependencies, timestamp)
```

Comment:
maybe a comment along the lines of "pods aren't considered in dependency calculations"